README.md

```diff
@@ -86,20 +86,20 @@ documentation of the latest stable version.
 
 ### Verifying Binaries
 
-Current, LTS and Nightly download directories all contain a *SHASUM256.txt*
+Current, LTS and Nightly download directories all contain a _SHASUM256.txt_
 file that lists the SHA checksums for each file available for
 download.
 
-The *SHASUM256.txt* can be downloaded using curl.
+The _SHASUM256.txt_ can be downloaded using curl.
 
-```
+```shell
```

Comment:
I disagree, but I'm open to be persuaded. Bike shed time, I guess but here are the reasons:

* This is a shell command. It may be bash. Or it maybe csh or zsh or tcsh or even Windows PowerShell. There is nothing bash-specific about `curl`. 

* If `shell` is not more appropriate than bash here, then when would it _ever_ be used?

* This is more of an argument that it doesn't matter than that one or the other is right, but [`bash` is an alias for `shell` in the syntax highlighting tool used by GitHub Flavored Markdown](https://github.com/github/linguist/blob/257425141d4e2a5232786bf0b13c901ada075f93/lib/linguist/languages.yml#L3532-L3539) so there is no difference in the rendered output of one versus the other.